docs/linux.md

```diff
@@ -39,15 +39,15 @@ Curl version 7.86
 	yum remove curl libcurl-devel
 	```
```

Comment:
DO we need to remove openssl-devel or libssl packages ?

Reply:
yes